packages/jest-console/src/__tests__/bufferedConsole.test.ts

```diff
@@ -40,7 +40,9 @@ describe('CustomConsole', () => {
       _console.assert(false);
 
       expect(stdout()).toMatch('AssertionError');
-      expect(stdout()).toMatch('false == true');
+      expect(stdout()).toMatch(
+        /false == true|The expression evaluated to a falsy value/,
```

Comment:
difference between node versions? I'm very surprised this is affected by the babel upgrade, tho

Reply:
https://github.com/facebook/jest/pull/13422#issuecomment-1273396392

Here's a more detailed explanation.
The normal behavior is that nodejs will output the code text. Previously, for some reason, nodejs could not successfully get the code text, and instead output `false !== true`.
Now nodejs seems to be able to get it, albeit in the wrong location. But it doesn't matter, because our assert is wrapped, nodejs can never output user code text normally.

Reply:
ah ok, cool!